test/kubemark/start-kubemark.sh

```diff
@@ -157,7 +157,7 @@ gcloud compute ssh --zone="${ZONE}" --project="${PROJECT}" "${MASTER_NAME}" \
     sudo bash -c \"echo ${password},admin,admin > /etc/srv/kubernetes/basic_auth.csv\""
 
 
-gcloud compute copy-files --zone="${ZONE}" --project="${PROJECT}" \
```

Comment:
See if it's actually working. I remember I had problems with ssh/copy-files at some point.